nemo/core/classes/exportable.py

```diff
@@ -48,7 +48,22 @@ def input_module(self):
     def output_module(self):
         return self
 
-    def export(
+    def export(self, output, input_example=None, **kwargs):
```

Comment:
Can the kwargs be written explicitly rather than **kwargs, autocomplete wont work otherwise

Reply:
You mean IDE function ? I doubt it's such a big deal, and spelling out and passing through all the parameters would look so much less 'pythonesque'.

Reply:
Not just IDE, but also for sphinx documentation, colab etc.

Reply:
Done.